utils/breakpoints.js

```diff
@@ -11,7 +12,7 @@ function dispatchActiveQuery(dispatch, mediaQueryState, action) {
     curr.matches ? curr : prev && prev.matches ? prev : null
   );
 
-  const breakpointName = activeQuery ? activeQuery.name : 'xlarge';
+  const breakpointName = activeQuery ? activeQuery.name : 'xxlarge';
```

Comment:
Does this work?  We have xxlarge, but only xlarge defined in the breakpoints list.  Also, we have 4 breakpoints as opposed to 5 originally?

Reply:
yep xxlarge will be the default one.